src/components/editor/prosemirror/module.ts

```diff
@@ -21,20 +23,39 @@ export { EditorView, EditorState, Plugin, defaultMarkdownSerializer };
 const schema = createSchema();
 const parser = createMarkdownParser(schema);
 
-export function createEditorState(opts: {content: ContentContainer }) {  
-    return EditorState.create({
-        doc: parser.parse(opts.content.textContent),
-        plugins: [
-            ...setupPlugins({ schema }),
-            new Plugin({
-                view: () => {
-                    return {
-                        update: debounce((view: EditorView) => {
-                            opts.content.textContent = defaultMarkdownSerializer.serialize(view.state.doc);
-                        }, 50)
-                    };
-                },
-            }),
-        ],
+export function createProseMirrorEditor(element: HTMLElement, cell: Cell, opts: { focusAfterInit?: boolean }, runtime: Runtime) {
```

Comment:
Nit: as the amount of arguments grows, I usually end up putting stuff more and more into an options object as the last argument, could you swap it to be the last? I usually also give it a default value of `= {}` if everything is optional.

As mentioned in the other comment, we can replace `cell` here with the more generic `something with a textContent field` type.